### PR TITLE
JK-104:  disable checkout btn until address been provied

### PR DESCRIPTION
**Pull request by dembar**
JK-104: Add validation on OrderSummary.jsx for disable checkout btn until address been provied

### CODE REVIEW

**Comment by sagun03** [comment]
image
 there is one more condition
Go to Checkout button should be disabled even if the address is present but its not select .

File: src/components/address/DataProvider.jsx [hunk 1]
```diff
@@ -73,7 +73,7 @@ const AddressProvider = ({ children }) => {
   const [selectedAddress, setSelectedAddress] = useState(null);
   const [defaultIndex, setDefaultIndex] = useState(null);
   const [submit, setSubmit] = useState(false)
-
+  // const [newAddress, setNewAddress] = useState(false);
```
Comment (sagun03) [inline src/components/address/DataProvider.jsx]: remove this commented code

File: src/components/address/DataProvider.jsx [hunk 2]
```diff
@@ -104,6 +104,8 @@ const AddressProvider = ({ children }) => {
         defaultIndex,
         setDefaultIndex,
         refetch
+        // newAddress,
+        // setNewAddress
```
Comment (sagun03) [inline src/components/address/DataProvider.jsx]: remove this commented code as well

**Review by git-nitin01 — APPROVED** [review]

**Review by git-nitin01 — COMMENTED** [review]
@dembar 
Reset the last commit, force push to delete the PR.
Make a new push with comments @sagun03 resolved and then make a new PR for merge approval to main branch.

**Pull request closed without merging**